qp_shotgun/qc_trim/qc_trim.py

```diff
@@ -161,14 +161,15 @@ def generate_qc_trim_commands(forward_seqs, reverse_seqs, map_file,
     """
     # we match filenames, samples, and run prefixes
     samples = make_read_pairs_per_sample(forward_seqs, reverse_seqs, map_file)
-	#threads = 4
+    #threads = 4
     cmds = []
 
     param_string = _format_qc_trim_params(parameters)
     for run_prefix, sample, f_fp, r_fp in samples:
         r_fp_str = ' --input "%s"' % r_fp if r_fp is not None else ""
```

Comment:
Note that the `-o` and `-p` flags in Atropos indicate the output file names. We should make them different from the input file names, perhaps by adding `_trimmed` to the filename prior to the extension?